.classpath

```diff
@@ -37,5 +37,8 @@
 	<classpathentry kind="lib" path="test-libs/junit-4.11.jar"/>
 	<classpathentry kind="lib" path="test-libs/hamcrest-core-1.3.jar"/>
 	<classpathentry kind="lib" path="test-libs/mockito-all-1.9.5.jar"/>
+	<classpathentry kind="lib" path="lib/deps/jackson-annotations-2.9.5.jar"/>
```

Comment:
We should use only one json library, there's already another one 4 lines up.

Reply:
I agree with this wholeheartedly, and my next project is to strip out org.json (both the seemingly unused vendored source code and the jar & uses in the scoreboard.  org.json has a weird license and is much lower level than we should really be using.

That said, I think removing org.json is a bit of an undertaking that is outside of the scope of this PR.  if you'd prefer me to rewrite PenaltyCodeManager to use org.json constructs for now I can, but I would rather not do that just to undo it in a few weeks.

Also, after doing a bit of reading (and realizing that I had added a 1.3MB dependency), I swapped jackson-databind for jackson-jr, that is much lighter weight but still gives us a much nicer org.json replacement.

Reply:
It's not the best of libraries alright.

> I would rather not do that just to undo it in a few weeks.

I'd hold off on that until I've finished removing the Game directory, as that's going to have json-y impacts. Does jackson-jr permit building up json by hand like the currently library does?

Reply:
yes, jackson-jr provides a fluent api that works very similarly to how org.json manipulates json objects.